src/main/scala/edu/ie3/simona/model/participant/HpModel.scala

```diff
@@ -7,7 +7,9 @@
 package edu.ie3.simona.model.participant
 
 import edu.ie3.datamodel.models.input.system.HpInput
-import edu.ie3.simona.agent.participant.data.Data.PrimaryData.ApparentPowerAndHeat
+import edu.ie3.simona.agent.participant.data.Data.PrimaryData.{
+  ApparentPowerAndHeat => ComplexPowerAndHeat
```

Comment:
Because I renamed `ApparentPower` `ComplexPower`. Maybe we could rename all occurrences of `ApparentPower` to `ComplexPower` where we have both the active and the reactive power. But this should be resolved in another issue.

Reply:
Saw that, but I was wondering we did not everywhere. Why not doing it within this PR? But before doing so, I would like to hear the opinion of @jo-bao and @sebastian-peter as well.

Reply:
I saw that, but I was wondering why you did not adapt everywhere. Is there a reason to adapt this in another issue/PR? But before doing so, I would like to hear the opinion of @jo-bao and @sebastian-peter as well.

Reply:
Is there a update for this?

Reply:
As I see, ApparentPower has been renamed to ComplexPower everywhere, so this should be solved?

Reply:
Not yet everywhere, but that would be the plan I guess. So ApparentPowerAndHeat should become ComplexPowerAndHeat

Reply:
Ah no, I was fooled by the import renamings. I'd rename the complex ApparentPower to ComplexPower (or ComplexApparentPower? @danielfeismann).

Reply:
ComplexPower = S, ApparentPower = |S|
So it should be ComplexPower imho.

Reply:
Is there an update on this discussion?

Reply:
What @danielfeismann said